src/Microsoft.VisualStudio.Threading/AsyncReaderWriterLock.cs

```diff
@@ -38,9 +38,9 @@ namespace Microsoft.VisualStudio.Threading
     public partial class AsyncReaderWriterLock : IDisposable
     {
         /// <summary>
-        /// A time delay to check whether pending writer lock and reader locks forms a dead lock.
+        /// A time delay to check whether pending writer lock and reader locks forms a deadlock.
         /// </summary>
-        private const int DefaultDeadLockCheckTimeOutInMilliseconds = 3 * 1000;
+        private static readonly TimeSpan DefaultDeadlockCheckTimeOut = new TimeSpan(hours: 0, minutes: 0, seconds: 3);
```

Comment:
tip: I find `TimeSpan.FromSeconds(3)` to be a succinct way of expressing this.

Reply:
Yeah, that is right :).